src/Core/SettingsChangesHistory.cpp

```diff
@@ -52,6 +52,8 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"azure_sdk_use_native_client", false, true, "New setting"},
             {"opentelemetry_trace_cpu_scheduling", false, false, "New setting to trace `cpu_slot_preemption` feature."},
             {"vector_search_with_rescoring", true, true, "New setting."},
+            {"s3_retry_scale_factor", 0, 25, "New setting."},
+            {"s3_retry_max_delay_ms", 0, 90000, "New setting."},
```

Comment:
Was it indeed behave as `0` before?

Reply:
It was `maxDelayMs_ = 5000`; Should we really make it higher?

Reply:
> Was it indeed behave as `0` before?

There weren't such settings before, so I made it as `0`. I think only this can pass the CI test?

> It was maxDelayMs_ = 5000; Should we really make it higher?

Oh, I followed the old default value (long time ago). I think 5000 is Ok, 90000 is too high.